theme/templates/email/email_verify.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+{% block main %}
+<p>{% trans "From HydroShare. This email address was used for updating your existing email on www.hydroshare.org." %}</p>
+<p>{% trans "If you originated the request, please use the link below to verify your email address to update your account email." %}</p>
+<p><a href="{{ request.scheme }}://{{ request.get_host }}{{ verify_url }}">{{ request.scheme }}://{{ request.get_host }}{{ verify_url}}</a></p>
```

Comment:
@mauriel133 I am bringing in @dtarb here to see whether he wants to change the email message for email verify when updating existing email from user profile page. @dtarb Since you finalized the email confirmation message for account signup, can you look at the email update verify message here to change wording/language as needed?